XandO.java

```diff
@@ -1,5 +1,104 @@
+import java.util.*;
+
 class PlayGame {
+    private static Scanner in = new Scanner(System.in);
     public static void main (String args[]) {
         XandO_Game xg = new XandO_Game();
+        startGame(xg);
+    }
+
+    // TODO: Move this into api class better better encapsulation.
+    // TODO: Break below function into more smaller logical components for simplicity
+    // TODO: Currently it is single time play per run, and can be played on single machine. Change that in future.
+    private static void startGame(XandO_Game xg) {
+        initialInstructions(xg);
+        int player = 1;
+        char move = selectPlayerOneChar();
```

Comment:
yes, that is definitely going to be important. Cant have this in current pull request as it is out of the scope, what I can do is add a TODO comment to do it in future.